source/computer/fedora-setup.rst

```diff
@@ -188,6 +188,8 @@ Intel 软件开发工具包
 
 更多设置可以参考\ `官方手册 <https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-intel-oneapi-base-linux/>`__\ 。
 
+.. include:: /computer/intel-oneAPI-warning.rst_
+
```

Comment:
Because it's in the current directory, you can simply use relative path. But I'm OK with the absolute path if you like.
```suggestion
.. include:: intel-oneAPI-warning.rst_
```